go.mod

```diff
@@ -10,16 +10,20 @@ require (
 	github.com/minio/md5-simd v1.1.2
 	github.com/minio/sha256-simd v1.0.1
 	github.com/rs/xid v1.5.0
-	github.com/sirupsen/logrus v1.9.3
+	github.com/stretchr/testify v1.8.4
```

Comment:
No testify just use Go's standard testing.

Reply:
I had done it since it's already pulled (a dep of `gopkg.in/ini.v1`) and thought we could use to simplify tests.
Instead reverted it and replaced the version in `go.mod`.